### ISSUE TITLE
Additional CSS navigation button only shown when there is some additional CSS already set

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#66333]
### Description

Noticed on a screenshot provided by @jameskoster on https://github.com/WordPress/gutenberg/issues/66307#issuecomment-2428946377 

Looking at that screenshot, I was wondering why I didn't see any 'Additional CSS' button in my Styles panel.

Turns out that seems intentional, as the navigation button [is rendered based on a condition](https://github.com/WordPress/gutenberg/blob/80fcf02a844454a812ce7f4070c2a4cac80d54f8/packages/edit-site/src/components/global-styles/screen-root.js#L118) that also checks whether some additional CSS is already set.

I had to figure out how to navigate to the Additional CSS panel and ultimately found it in the 'More' ellipsis button at the top of the Styles panel.

To me, this seems to be a perfect way to hide features and make them less discoverable from users. I'd appreciate to know why the navigation button is rendered conditionally and whether this makes the UI more usable in any way.

I'd like to propose to make things simple and just not implement behaviors that may look smart at first but ultimately are arguable ad likely confusing for users.

@WordPress/gutenberg-design do you know why this is rendered only _after_ some adidtional CSS has been set?

Also, the feature description:
`Add your own CSS to customize the appearance and layout of your site.`
is completely missing at first and only shown _after_ the feature has been used. It's arguably usable to provide a description only _after_ the feature it relates too has been used.

![Image](https://github.com/user-attachments/assets/d2a2db94-18fd-432f-b242-a0efc93d1d59)

### Step-by-step reproduction instructions

- Make sure you dont have any additional CSS set in the Global Styles.
- Open the Styles panel
- Observe there is no 'Additional CSS' navigation button at the bottom of the panel.
- Open the ellipsis menu and click 'Additional CSS'.
- Enter some additional CSS.
- No need to save.
- Go back to the main panel by using the 'Back'  chevron icon button at the top of the additional CSS panel.
- Onserve the main Styles panel and see there is now an 'Additional CSS' navigation button, together with the feature description.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by afercia** [comment]
Additionally:
Why the 'Blocks' and 'Additional CSS' descriptions are placed _before_ the navigation buttons? 
This seems inconsistent with the vast majority of cases where descriptions in the editor are placed _after_ the related control.

Also, for better accessibility, these descriptions should be associated with their related controls via `aria-describedby`.

**Comment by t-hamano** [comment]
>do you know why this is rendered only after some adidtional CSS has been set?

It seems related to #47266.

I too would expect Additional CSS button to always be rendered. This functionality has been present in the Customizer for a long time and I don't think it should be hidden by default:

![Image](https://github.com/user-attachments/assets/9994a903-faa0-4c9f-bd5a-e15e596c7115)

**Comment by afercia** [comment]
@t-hamano thanks for pointing to https://github.com/WordPress/gutenberg/issues/47266

It appears the two points in that issues are:
> Move the Additional CSS access point to the Global Styles ellipsis menu
> Only if there is additional CSS present, add the Additional CSS component top-level to Global Styles (like it is currently)

i disagree with both points. To me, initially burying down a feature into a barely noticeable ellipsis menu is a perfect way to hide that feature. The fact the navigation button appeaers only when there is some additional CSS set may be smart at first but it;s confusing for users. I can't support this kind of design approach where, in the name of aiming to make the UI 'less cluttered', features and content get removed or hidden. 

I only share the point that Additional CSS shouldn't be much prominent. It's sort of an advanced feature but it still needs to be discoverable. I'd think the best option is to just place the navigation button after the 'Blocks' one, with no special prominence.

**Comment by afercia** [comment]
It's worth mentioning that based on https://github.com/WordPress/gutenberg/pull/63243#issuecomment-2422638895 and https://github.com/WordPress/gutenberg/issues/66307 the Revisions button is candiate to be moved to the main panel as well. So taht there would be 3 navigation buttons always shown, with no special prominence:

![Image](https://github.com/user-attachments/assets/9d809b35-f1f8-4b7c-83eb-29642f59cfb2)

**Comment by afercia** [comment]
Looking into this, I just noticed a different beheavior when accessing the Additional CSS that I'd consider a functional bug.
- Make sure you have some additional CSS set e.g. a simple `p {background: red; }`
- Click the Style Book. The editor canvas changes to show the style book view.
- Access the Additional CSS from the ellipsis menu at the top of the Styles panel.
- Observe the canvas view changes and switches back to the default view.
- Click Style Book again.
- Access the Additional CSS from the navigatiom button at the bottom of the Styles panel.
- Observe the canvas view _does not_ change.

![Image](https://github.com/user-attachments/assets/442ea39c-06a1-43fb-be69-a3e64574af5b)

**Comment by nukak2024** [comment]
While working with a user, I encountered this issue where the _Additional CSS_ option was unexpectedly missing from the _Appearance > Editor > Styles_ panel when using the **Twenty Twenty-Five** theme. Interestingly, when switching to the **Twenty Twenty-Four** theme, the _Additional CSS_ section appeared as expected.

This caused some confusion for the user, as they expected the _Additional CSS_ option to be consistently available across block themes, especially when switching between default WordPress themes. 

Screencast: https://d.pr/v/tNnVqB

**Comment by dsas** [comment]
@nukak2024's video shows that this has become more confusing now that we also have the left styles interface which doesn't have a three dot menu with Additional CSS inside.

**Comment by jhmonroe** [comment]
+1 for not making this a default. It's totally confusing and creates a situation where only deep Wordpress people in the know can understand, find, or use this. Runs contrary to democratizing publishing

**Comment by t-hamano** [comment]
Update: We changed the Additional CSS button to always be visible in #71537.

Let's close this issue and continue the discussion about improving the Style Panel itself in #66307.

**Comment by jhmonroe** [comment]
@t-hamano you're the GOAT! People like you are why I keep coming back to contribute in the little ways I can as a designer/user/front-end tinkerer
